platformio/commands/account/command.py

```diff
@@ -0,0 +1,202 @@
+    client = AccountClient()
+    try:
+        client.change_password(old_password, new_password)
+        return click.secho("Password successfully changed!", fg="green")
+    except exception.AccountNotAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
+
+@cli.command("token", short_help="Get or regenerate Authentication Token")
+@click.option("-p", "--password", prompt=True, hide_input=True)
+@click.option("--regenerate", is_flag=True)
+@click.option("--json-output", is_flag=True)
+def account_token(password, regenerate, json_output):
+    client = AccountClient()
+    try:
+        auth_token = client.auth_token(password, regenerate)
+        if json_output:
+            return click.echo(json.dumps({"status": "success", "result": auth_token}))
+        return click.secho("Personal Authentication Token: %s" % auth_token, fg="green")
+    except exception.AccountNotAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
+
+@cli.command("forgot", short_help="Forgot password")
+@click.option("--username", prompt="Username or e-mail")
+def account_forgot(username):
+    client = AccountClient()
+    try:
+        client.forgot_password(username)
+        return click.secho(
+            "If this account is registered, we will send the "
+            "further instructions to your E-Mail.",
+            fg="green",
+        )
+    except exception.AccountAlreadyAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
+
+@cli.command("update", short_help="Update profile information")
+@click.option("--current-password", prompt=True, hide_input=True)
+def account_update(current_password):
+    client = AccountClient()
+    try:
+        profile = client.get_profile()
+        new_profile = {}
+        for field in profile:
+            new_profile[field] = click.prompt(
+                field.replace("_", " ").capitalize(), default=profile[field]
+            )
+            if field == "email":
+                validate_email(new_profile[field])
+            if field == "username":
+                validate_username(new_profile[field])
+        client.update_profile(new_profile, current_password)
+        if new_profile["email"] != profile["email"]:
+            try:
+                client.logout()
+            except exception.AccountNotAuthenticated:
+                pass
+            return click.secho(
+                "Profile successfully updated! "
+                "Please check your mail to verify your new email address and re-login.",
+                fg="green",
+            )
+        if new_profile["username"] != profile["username"]:
+            try:
+                client.logout()
+            except exception.AccountNotAuthenticated:
+                pass
+            return click.secho(
+                "Profile successfully updated! Please re-login.", fg="green"
+            )
+        return click.secho("Profile successfully updated!", fg="green")
```

Comment:
click.secho("Profile successfully updated!", fg="green")
username_changed = new_profile["username"] != profile["username"]
email_changed = new_profile["email"] != profile["email"]
if not username_changed and not email_changed:
     return

            try:
                client.logout()
            except exception.AccountNotAuthenticated:
                pass

if email_changed:
   return click.secho("Please check your mail to verify your new email address and re-login. ", fg="yellow")
click.secho("Please re-login.", fg="yellow")